app/keepers/keepers.go

```diff
@@ -664,15 +679,21 @@ func (ak *AppKeepers) InitKeepers(
 
 	// Create Transfer Stack
 	// SendPacket Path:
-	// SendPacket -> Transfer -> Callbacks -> Fee -> IBC core (ICS4Wrapper)
+	// SendPacket -> Transfer -> Callbacks -> PFM -> Fee -> IBC core (ICS4Wrapper)
 	// RecvPacket Path:
-	// RecvPacket -> IBC core -> Fee -> Callbacks -> Transfer (AddRoute)
+	// RecvPacket -> IBC core -> Fee -> PFM -> Callbacks -> Transfer (AddRoute)
 	// Receive path should mirror the send path.
 	var transferStack porttypes.IBCModule
 	transferStack = transfer.NewIBCModule(ak.TransferKeeper)
 	cbStack := ibccallbacks.NewIBCMiddleware(transferStack, ak.IBCFeeKeeper, wasmStackIBCHandler,
```

Comment:
for send path to be: `SendPacket -> Transfer -> Callbacks -> PFM -> Fee -> IBC core (ICS4Wrapper)`

Shouldn't `ak.IBCFeeKeeper` be replaced with `ak.PFMRouterKeeper` ?  (checkout Gaia: https://github.com/cosmos/gaia/blob/release/v23.x/app/keepers/keepers.go#L521)

Reply:
good catch! Thanks!
Updated with the change